core/src/main/java/org/springframework/security/oauth2/server/authorization/authentication/OAuth2ClientAuthenticationToken.java

```diff
@@ -23,6 +23,7 @@
 
 /**
  * @author Joe Grandja
+ * @author Patryk Kostrzewa
  */
 public class OAuth2ClientAuthenticationToken extends AbstractAuthenticationToken {
```

Comment:
For constructor arguments, apply `Assert.notNull()` or `Assert.hasText()` or whatever makes sense.

For private members, mark `final` whenever applicable (set in constructor and no corresponding setter)